content/docs/command-reference/status.md

```diff
@@ -157,44 +157,45 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
-        changed outs:
-                not in cache:      foo
-foo.dvc:
-        changed outs:
-                deleted:      foo
-        changed checksum
-prepare.dvc:
-        changed outs:
-                new:      bar
-        always changed
+dofoo:
+	changed deps:
+		modified:           bar
+	changed outs:
+		not in cache:       foo
+dobar:
+	changed outs:
+		deleted:            bar
+	changed checksum
+baz.dvc:
+	changed outs:
+		new:                baz
+	always changed
 ```
 
-This shows that for stage `bar.dvc`, the dependency `foo` and the
-<abbr>output</abbr> `bar` have changed. Likewise for `foo.dvc`, the dependency
-`foo` has changed, but no output has changed.
+This shows that for stage `foo`, the dependency `bar` and the
+<abbr>output</abbr> `foo` have changed, and for stage `bar`, the output `bar`
+has changed, but no dependency has changed. Likewise for `baz.dvc`, the output
+`baz` tracked by it has changed.
 
 ## Example: Specific files or directories
 
 `dvc status` only checks the tracked data corresponding to any given `targets`:
 
 ```dvc
-$ dvc status foo.dvc dobar
+$ dvc status dofoo.dvc dobar
 foo.dvc:
-  changed outs:
-          deleted:      foo
-  changed checksum
+	changed outs:
+		deleted:            foo
+	changed checksum
 dobar:
-  changed deps:
-          modified:      bar
-  changed outs:
-          not in cache:      foo
+	changed deps:
+		modified:           bar
+	changed outs:
+		not in cache:       foo
```

Comment:
Same here. How can foo be an output of both foo.dvc and dobar? Overlapping outputs are not allowed.

Reply:
Interchanged `foo` and `bar` here. So `foo` can be an outs of `.dvc` file as well as deps of a stage.

Reply:
Hmmmm now foo is deleted in foo.bar but modified in dobar. Not consistent. Again, let's actually build the dvc.yaml and run `status`. We can't make these up by hand, there will be mistakes. Thanks

Reply:
Updated this.

Reply:
Updated this.